dScripts/ai/MINIGAME/SG_GF/SERVER/SGCannon.cpp

```diff
@@ -76,7 +76,7 @@ void SGCannon::OnActivityStateChangeRequest(Entity* self, LWOOBJID senderID, int
 		auto* player = EntityManager::Instance()->GetEntity(self->GetVar<LWOOBJID>(PlayerIDVariable));
 		if (player != nullptr) {
 			Game::logger->Log("SGCannon", "Player is ready");
-			/*GameMessages::SendSetStunned(player->GetObjectID(), PUSH, player->GetSystemAddress(), LWOOBJID_EMPTY,
+			/*GameMessages::SendSetStunned(player->GetObjectID(), eStateChangeType::PUSH, player->GetSystemAddress(), LWOOBJID_EMPTY,
```

Comment:
Why are we keeping this code?  Is this in the live script?

Reply:
I found and replaced that variable so that's why it changed, and yes it's in the live script.